Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -347,4 +349,25 @@ else if (elementKind.equals(ElementKind.TOP_LEVEL_VARIABLE)) {
       return null;
     }
   }
+
+  @Override
+  public void beforeCompletion(@NotNull final CompletionInitializationContext context) {
```

Comment:
As far as I see this method has been moved without any changes inside. Has it been done on purpose? I'd prefer to avoid such changes as they pollute code annotation (I mean right-click gutter -> Annotate) and make it harder to track file history.